### ISSUE TITLE
Difference between `JetBrainsMono-NF` and `JetBrainsMono-NF-Mono` is unclear

### ISSUE THREAD

**Issue by Fleshgrinder** [matthewjberger/scoop-nerd-fonts#205]
An innocent `scoop search jetbrainsmono` provides `JetBrainsMono-NF` and `JetBrainsMono-NF-Mono` but it is not clear what the difference is between these two. Using `scoop info` provides the exact same description for both. Comparing their manifests side-by-side results in the following diff:

```diff
--- bucket/JetBrainsMono-NF.json
+++ bucket/JetBrainsMono-NF-Mono.json
-             "Get-ChildItem $dir -Filter '*Complete Windows Compatible.*' | ForEach-Object {",
+             "Get-ChildItem $dir -Filter '*Mono Windows Compatible.*' | ForEach-Object {",
```

It is still not clear what the difference is though. I suspect that `JetBrainsMono-NF` is with ligatures and `JetBrainsMono-NF-Mono` is not?!? 😕

I think that the packages need to be renamed so that they are clearer, and their descriptions needs to properly state what they provide.

**Comment by DenisionSoft** [comment]
@Fleshgrinder 
See [nerd-fonts#945](https://github.com/ryanoasis/nerd-fonts/discussions/945) for NF vs NFM and [nerd-fonts#891](https://github.com/ryanoasis/nerd-fonts/discussions/891) for NF vs Nerd Font.

TL;DR - in NFM symbols fit in one cell, in regular NF they don't, which makes them bigger. NF vs Nerd Font is just a _possible_ Windows limitation issue.

Also, the no ligatures version has `NL` ending (like in `JetBrainsMonoNL`).

**Comment by jonz94** [comment]
Hi @Fleshgrinder,

I send a PR to add descriptions for these fonts, this is the result will look like:

![result](https://user-images.githubusercontent.com/16042676/202879747-71fa1f87-708d-405e-8b87-9792a502b71a.png)

**Comment by jonz94** [comment]
> I think that the packages need to be renamed so that they are clearer

This is a good idea, but unfortunately there is a caveat when we change the package name.

Currently, scoop do not have a way to detect if a package has been rename to something else, and scoop will assume that the old package has been deleted.

For example, if we change `JetBrainsMono-NF` to `JetBrainsMono-Nerd-Font-Normal`, users who have installed `JetBrainsMono-NF` will only get a message tell them "JetBrainsMono-NF has been removed"

![scoop-status](https://user-images.githubusercontent.com/16042676/202880370-ab2761ae-786c-4f44-9d99-793f85a36393.png)

They might have no idea what happens until they come to GitHub and see that the package name has been change to `JetBrainsMono-Nerd-Font-Normal`.

So, I think we should not change the package name for now.

**Comment by Fleshgrinder** [comment]
The description is the important part. 😊
